src/main/webapp/app/shared/sidebar/sidebar.component.ts

```diff
@@ -36,15 +46,20 @@ export class SidebarComponent implements OnDestroy, OnChanges, OnInit {
     isProduction = true;
     isTestServer = false;
 
-    // icons
-    faMagnifyingGlass = faMagnifyingGlass;
-    faChevronRight = faChevronRight;
-    faFilter = faFilter;
+    private modalRef: NgbModalRef | null;
```

Comment:
is null here used because of any conditions of NgbModal? (And not undefined or like the following)
```suggestion
    private modalRef?: NgbModalRef;
```